scripts/csv-to-json/index.ts

```diff
@@ -1,18 +1,26 @@
 import * as fs from 'fs'
 
 // TODO function that calls writeJsonFromCsv for everything in ./csv
+function writeAll(dirPath: string) {
+  fs.readdirSync(dirPath).forEach((fn: string) => {
+    writeJsonFromCsv(dirPath, fn)
+  })
+}
 
-function writeJsonFromCsv(fileName: string) {
-  const json = csvToType(loadCsv(fileName))
+
+function writeJsonFromCsv(dirPath: string, fileName: string) {
+  console.log('trying to write', fileName)
+  const json = csvToType(loadCsv(dirPath, fileName))
   console.log(json)
   fs.writeFileSync(
-    `${__dirname}/json/${fileName}.json`, 
+    `${__dirname}/json/${fileName.slice(0, fileName.length - 4)}.json`,
```

Comment:
This feels fragile

Reply:
This whole pipeline feels pretty fragile to me, but this doesn't seem like a major offender: it's just chopping off ".csv." Do you think we should change it?

Reply:
aah, I think there are things that are much more direct and clear we can use.

Like node's path library. https://nodejs.org/api/path.html#pathbasenamepath-ext
or js built in string replace https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace

but ultimately I think we should use a library that parses csv to JSON like this https://www.npmjs.com/package/csv-parser